Blockzilla/SettingsViewController.swift

```diff
@@ -180,18 +169,37 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
     private var highlightsButton: UIBarButtonItem?
     private let whatsNew: WhatsNewDelegate
     private lazy var sections = {
-        Section.getSections(deviceHasBiometrics: biometryType.hasBiometry)
+        Section.getSections()
     }()
-    
-    private var toggles = [
-        BlockerToggle(label: UIConstants.strings.toggleSafari, setting: SettingsToggle.safari),
-        BlockerToggle(label: UIConstants.strings.labelBlockAds, setting: SettingsToggle.blockAds, subtitle: UIConstants.strings.labelBlockAdsDescription),
-        BlockerToggle(label: UIConstants.strings.labelBlockAnalytics, setting: SettingsToggle.blockAnalytics, subtitle: UIConstants.strings.labelBlockAnalyticsDescription),
-        BlockerToggle(label: UIConstants.strings.labelBlockSocial, setting: SettingsToggle.blockSocial, subtitle: UIConstants.strings.labelBlockSocialDescription),
-        BlockerToggle(label: UIConstants.strings.labelBlockOther, setting: SettingsToggle.blockOther, subtitle: UIConstants.strings.labelBlockOtherDescription),
-        BlockerToggle(label: UIConstants.strings.labelBlockFonts, setting: SettingsToggle.blockFonts),
-        BlockerToggle(label: UIConstants.strings.labelSendAnonymousUsageData, setting: SettingsToggle.sendAnonymousUsageData),
-    ]
+
+    private var toggles = [Int : BlockerToggle]()
+
+    private var initialToggles : [Int : BlockerToggle]  {
+        let blockFontsToggle = BlockerToggle(label: UIConstants.strings.labelBlockFonts, setting: SettingsToggle.blockFonts)
+        let toggle = blockFontsToggle.toggle
+        toggle.onTintColor = UIConstants.colors.toggleOn
+        toggle.tintColor = UIConstants.colors.toggleOff
```

Comment:
Are we able to make these default values inside the initialization of a BlockerToggle or do we want toggles to not appear this way more often?